testnets/_IBC/xiontestnet-axelartestnet.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "xiontestnet",
```

Comment:
do avoid duplicate files for the same connection, the chain_names are sorted alphabetically in both the name of the file and in the usage of chain_1 and chain_2, so xiontestnet must be chain_2 in all these cases

Reply:
Fixed the alpha ordering of the files and the chain entries. Thanks!